### ISSUE TITLE
Error with neovim

### ISSUE THREAD

**Issue by karambaq** [olimorris/dotfiles#7]
Hello, I've tried to copy your neovim config and facing following problems, could you help me resolve them?

1) 
```
Failed loading Oli.config.options
/Users/maksim/.config/nvim/lua/Oli/config/options.lua:19: attempt to concatenate a nil value
# stacktrace:
  - lua/Oli/config/options.lua:19
  - lua/Oli/plugins/config.lua:4
  - lua/Oli/plugins/config.lua:4 _in_ **load**
  - lua/Oli/plugins/config.lua:17
  - lua/Oli/config/lazy.lua:14
  - init.lua:6
Lua module not found for config of persisted.nvim. Please use a `config()` function instead
```

Hitting enter...

2) 
```
Failed to run `config` for onedarkpro.nvim
/Users/maksim/.config/nvim/lua/Oli/plugins/colorscheme.lua:214: module 'onedarkpro' not found:
^Ino field package.preload['onedarkpro']
cache_loader: module onedarkpro not found
cache_loader_lib: module onedarkpro not found
^Ino file './onedarkpro.lua'
^Ino file '/usr/local/share/luajit-2.1.0-beta3/onedarkpro.lua'
^Ino file '/usr/local/share/lua/5.1/onedarkpro.lua'
^Ino file '/usr/local/share/lua/5.1/onedarkpro/init.lua'
^Ino file './onedarkpro.so'
^Ino file '/usr/local/lib/lua/5.1/onedarkpro.so'
^Ino file '/usr/local/lib/lua/5.1/loadall.so'
# stacktrace:
  - lua/Oli/plugins/colorscheme.lua:214 _in_ **config**
  - lua/Oli/config/lazy.lua:14
 ```

3) 
```
Failed to run `config` for heirline.nvim
...rs/maksim/.config/nvim/lua/Oli/plugins/heirline/init.lua:74: module 'onedarkpro.helpers' not found:
^Ino field package.preload['onedarkpro.helpers']
cache_loader: module onedarkpro.helpers not found
cache_loader_lib: module onedarkpro.helpers not found
^Ino file './onedarkpro/helpers.lua'
^Ino file '/usr/local/share/luajit-2.1.0-beta3/onedarkpro/helpers.lua'
^Ino file '/usr/local/share/lua/5.1/onedarkpro/helpers.lua'
^Ino file '/usr/local/share/lua/5.1/onedarkpro/helpers/init.lua'
^Ino file './onedarkpro/helpers.so'
^Ino file '/usr/local/lib/lua/5.1/onedarkpro/helpers.so'
^Ino file '/usr/local/lib/lua/5.1/loadall.so'
^Ino file './onedarkpro.so'
^Ino file '/usr/local/lib/lua/5.1/onedarkpro.so'
^Ino file '/usr/local/lib/lua/5.1/loadall.so'
# stacktrace:
  - lua/Oli/plugins/heirline/init.lua:74 _in_ **load**
  - lua/Oli/plugins/heirline/init.lua:139 _in_ **config**
  - lua/Oli/config/lazy.lua:14
```

**Comment by olimorris** [comment]
Ah I suspect that's because I load them locally. Remove the following from `lazy.lua`:

```lua
dev = {
    path = "~/Code/Neovim",
    patterns = { "olimorris" },
  },
```

**Comment by trans4jp** [comment]
Hi i wanna understand heirline cofig writing so i tried and read ur config :)
but same error too

![swappy-20230328_224440](https://user-images.githubusercontent.com/127837675/228329895-81544a9e-bafd-4f73-ae37-7626a572bed0.png)

**Comment by olimorris** [comment]
@trans4jp you need to be on nightly to get the statuscolumn. Shouldn't be a painful update

**Comment by olimorris** [comment]
@trans4jp this is fixed as per 1b274a7
